lib/docker.js

```diff
@@ -107,6 +107,15 @@ function promisedSpawn(args, options) {
 
 }
 
+function generateCustomSourcesKeyFetch(customSourcePkgs) {
+    customSourcePkgs = customSourcePkgs.filter(function(pkgSpec) { return pkgSpec.key_url; });
+    if (customSourcePkgs.length) {
+        return 'RUN ' + customSourcePkgs.map(function(pkgSpec) {
+                return ' wget -O key "' + pkgSpec.key_url + '" && apt-key add key && rm key ';
```

Comment:
It could, but since we are doing this inside a container and already need to use temporary files and caches for fetching and installing DEBs anyway, I don't think this matters in practice.